plugins/outputs/zabbix/zabbix.go

```diff
@@ -0,0 +1,164 @@
+package zabbix
+
+import (
+	_ "embed"
+	"sort"
+	"time"
+
+	"github.com/datadope-io/go-zabbix"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+// Zabbix allows pushing metrics to Zabbix software
+type Zabbix struct {
+	Host                  string
+	Port                  int
+	AgentActive           bool `toml:"agent_active"`
+	Prefix                string
+	SkipMeasurementPrefix bool `toml:"skip_measurement_prefix"`
+
+	// Config for LLD and autoregister. Needed here for config parsing.
+	LLDPeriod              int `toml:"lld_period"`
+	LLDResetPeriod         int `toml:"lld_reset_period"`
+	Autoregister           string
+	AutoregisterSendPeriod config.Duration
```

Comment:
Please add TOML tags to all configuration settings! Furthemore, use a `config.Duration` for all periods...

Reply:
Fixed in 37d78aa67